pyomo/repn/plugins/mps.py

```diff
@@ -515,10 +515,22 @@ def yield_all_constraints():
         column_template = "     %s %s %" + self._precision_string + "\n"
         output_file.write("COLUMNS\n")
         cnt = 0
+        set_integer = False
         for vardata in variable_list:
             col_entries = column_data[variable_to_column[vardata]]
             cnt += 1
             if len(col_entries) > 0:
+                if vardata.is_integer() and not set_integer:
+                    set_integer = True
+                    output_file.write(
+                        "     %s %s %s\n" % ("INT", "'MARKER'", "'INTORG'")
+                    )
+                if not vardata.is_integer() and set_integer:
+                    set_integer = False
+                    output_file.write(
+                        "     %s %s %s\n" % ("INTEND", "'MARKER'", "'INTEND'")
+                    )
```

Comment:
Is it an issue that the marker name is hard-coded? It looks like 
- Gurobi [explicitly ignores](https://www.gurobi.com/documentation/current/refman/mps_format.html) the marker names
- Cplex requires the marker name to [differ](https://www.ibm.com/docs/en/icos/22.1.0?topic=extensions-integer-variables-in-mps-files) from the preceeding / subsequent variable names (and says nothing about uniqueness, although the example hints that uniqueness might be required?).  Note that a marker name could conflict with a column name, causing issues with cplex.
- Mosek [recommends markers](https://docs.mosek.com/latest/cxxfusion/mps-format.html#integer-variables) NOT be used.  They include it for backwards compatibility and ignore the name field.
- Xpress [does not comment ](https://www.fico.com/fico-xpress-optimization/docs/latest/solver/nonlinear/HTML/chapExtMPSformat.html) on the use of markers
- Lpsolve [requires](https://lpsolve.sourceforge.net/5.5/mps-format.htm) a name but does not make any statement as to its uniqueness

In addition, will solvers accept duplicate marker names if there are multiple groups of integer variables?